docs/doxygen-awesome-css/README.md

```diff
@@ -0,0 +1,6 @@
+Files in this subdirectory are from the [Doxygen Awesome][1] CSS theme
```

Comment:
I know you agonized over theme stuff, but as of 1.14.0 (2025-05-24) Doxygen itself has a much more modern theme. [See example here](https://doxygen.github.io/doxygen-docs/).

It might make more sense to just use it instead as that probably makes more sense for long-term maintenance. (Which I think is especially important if we start putting Doxygen on the individual device repos as well.)

Reply:
Agreed this does look almost identical.

I would be willing to consider changing to it now, the one snag I noticed is that although the new Doxygen docs website does have a light / dark theme switcher, this doesn't seem to be included when I build with 1.14.0, perhaps a configuration option that I missed?

Their footer does say "Generated with doxygen 1.15.0" which is not out yet, however I wasn't able to track down where this is implemented yet.

Reply:
Nevermind, found it `HTML_COLORSTYLE   = TOGGLE`.

Reply:
I still find everything a bit too small compared with the Doxygen Awesome CSS theme, and not particularly in love that the navigation header gets moved to an empty space in between.

I don't think we will start embedding Doxygen docs in individual devices. While the firmware will indeed be implemented in C/C++ I don't feel the C implementation needs to be surfaced to the reference documentation of each device, which would also let us keep each device docs in DocFX instead.

Reply:
We could definitely tweak the CSS a bit, but given you don't see us using Doxygen for individual devices it's not too important. Could always switch later if the Doxygen Awesome theme goes unmaintained.

> I don't feel the C implementation needs to be surfaced to the reference documentation of each device

(Also I agree, I just thought you mentioned it as something you wanted to do.)